translations/kubectl/de_DE/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3191 @@
+"resource=pods --resource-name=readablepod"
+msgstr ""
+"\n"
+"\t\t# Erstellt eine ClusterRole \"pod-reader\", die es Nutzern erlaubt "
+"\"get\", \"watch\" und \"list\" auf den Pods auszuführen\n"
+"\t\tkubectl create clusterrole pod-reader --verb=get,list,watch --"
+"resource=pods\n"
+"\n"
+"\t\t# Erstellt eine ClusterRole \"pod-reader\" mit dem angegebenen "
+"ResourceName\n"
+"\t\tkubectl create clusterrole pod-reader --verb=get,list,watch --"
+"resource=pods --resource-name=readablepod"
+
+#: pkg/kubectl/cmd/create_role.go:41
+msgid ""
+"\n"
+"\t\t# Create a Role named \"pod-reader\" that allows user to perform \"get"
+"\", \"watch\" and \"list\" on pods\n"
+"\t\tkubectl create role pod-reader --verb=get --verb=list --verb=watch --"
+"resource=pods\n"
+"\n"
+"\t\t# Create a Role named \"pod-reader\" with ResourceName specified\n"
+"\t\tkubectl create role pod-reader --verb=get --verg=list --verb=watch --"
+"resource=pods --resource-name=readablepod"
+msgstr ""
+"\n"
+"\t\t# Erstellt eine Role \"pod-reader\", die es dem Nutzer erlaubt "
+"\"get\", \"watch\" und \"list\" auf den Pods auszuführen\n"
+"\t\tkubectl create role pod-reader --verb=get --verb=list --verb=watch --"
+"resource=pods\n"
+"\n"
+"\t\t# Erstellt eine Role \"pod-reader\" mit dem angegebenen ResourceName\n"
+"\t\tkubectl create role pod-reader --verb=get --verg=list --verb=watch --"
+"resource=pods --resource-name=readablepod"
+
+#: pkg/kubectl/cmd/create_quota.go:35
+msgid ""
+"\n"
+"\t\t# Create a new resourcequota named my-quota\n"
+"\t\tkubectl create quota my-quota --hard=cpu=1,memory=1G,pods=2,services=3,"
+"replicationcontrollers=2,resourcequotas=1,secrets=5,"
+"persistentvolumeclaims=10\n"
+"\n"
+"\t\t# Create a new resourcequota named best-effort\n"
+"\t\tkubectl create quota best-effort --hard=pods=100 --scopes=BestEffort"
+msgstr ""
+"\n"
+"\t\t# Erstellt eine neue ResourceQuota my-quota\n"
+"\t\tkubectl create quota my-quota --hard=cpu=1,memory=1G,pods=2,services=3,"
+"replicationcontrollers=2,resourcequotas=1,secrets=5,"
+"persistentvolumeclaims=10\n"
+"\n"
+"\t\t# Erstellt eine neue ResourceQuota best-effort\n"
+"\t\tkubectl create quota best-effort --hard=pods=100 --scopes=BestEffort"
+
+#: pkg/kubectl/cmd/create_pdb.go:35
+#, c-format
+msgid ""
+"\n"
+"\t\t# Create a pod disruption budget named my-pdb that will select all pods "
+"with the app=rails label\n"
+"\t\t# and require at least one of them being available at any point in "
+"time.\n"
+"\t\tkubectl create poddisruptionbudget my-pdb --selector=app=rails --min-"
+"available=1\n"
+"\n"
+"\t\t# Create a pod disruption budget named my-pdb that will select all pods "
+"with the app=nginx label\n"
+"\t\t# and require at least half of the pods selected to be available at any "
+"point in time.\n"
+"\t\tkubectl create pdb my-pdb --selector=app=nginx --min-available=50%"
+msgstr ""
+"\n"
+"\t\t# Erstellt ein Pod-Disruption-Budget my-pdb, dass alle Pods mit "
```

Comment:
Sometimes the object names are quoted, sometimes they are not.

Reply:
Yes, but it's the same in the english template. I tried to stay with the original instead of being consistent. It's handled the same way in the other translations.
This is also the case for ReplicationController vs Replication-Controller vs Replication Controller etc.

If a consistent translation is preferred I can also quote the object names.